### DISCUSSION TITLE
v2.0 Inference

### DISCUSSION THREAD

**Discussion by rnaidu02** [mlcommons/inference#1134] [General]
v2.0 Inference 

<hr /><em>This discussion was created from the release <a href='https://github.com/mlcommons/inference/releases/tag/v2.0'>v2.0 Inference</a>.</em>

**Comment by TracyCAlbers** [comment]
Hello!  

I'm looking for a specific of code in LoadGen line where it sends inference requests to an API of your choice.  Can any one point me in the right direction?  

Thank you!

**Reply by nvpohanh** [reply]
Here? https://github.com/mlcommons/inference/blob/master/loadgen/issue_query_controller.cc#L416
